.github/workflows/main.yml

```diff
@@ -8,70 +8,80 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [py36, py37, py38, py39, py310]
+        name: ["py36", "py37", "py38", "py39", "py310", "py311-dev"]
         include:
-        - name: py36
-          python: 3.6
+        - python: "3.6"
           docutils: du14
-        - name: py37
-          python: 3.7
+        - python: "3.7"
           docutils: du15
-        - name: py38
-          python: 3.8
+        - python: "3.8"
           docutils: du16
-        - name: py39
-          python: 3.9
+        - python: "3.9"
           docutils: du17
-          coverage: "--cov ./ --cov-append --cov-config setup.cfg"
-        - name: py310
-          python: "3.10"
-          docutils: du17
-        - name: py311-dev
-          python: 3.11-dev
+        - python: "3.10"
+          docutils: du18
+        - python: "3.11-dev"
           docutils: py311
-    env:
-      PYTEST_ADDOPTS: ${{ matrix.coverage }}
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python }}
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v3
       if: "!endsWith(matrix.python, '-dev')"
       with:
         python-version: ${{ matrix.python }}
     - name: Set up Python ${{ matrix.python }} (deadsnakes)
       uses: deadsnakes/action@v2.0.1
-      if: endsWith(matrix.python, '-dev')
+      if: "endsWith(matrix.python, '-dev')"
       with:
         python-version: ${{ matrix.python }}
     - name: Check Python version
       run: python --version
     - name: Install graphviz
       run: sudo apt-get install graphviz
     - name: Install dependencies
-      run: pip install -U tox codecov
-    - name: Install the latest py package (for py3.11-dev)
-      run: pip install -U git+https://github.com/pytest-dev/py
-      if: ${{ matrix.python == '3.11-dev' }}
+      run: python -m pip install -U tox pip
     - name: Run Tox
       run: tox -e ${{ matrix.docutils }} -- -vv
-    - name: codecov
-      uses: codecov/codecov-action@v1
-      if: matrix.coverage
 
   windows:
     runs-on: windows-2019
-    strategy:
-      matrix:
-        architecture: [x86, x64]
```

Comment:
Is x86 no longer needed?

Note: I don't have opinion for this change because I'm not sure about Windows at all.

Reply:
I removed it to reduce CI resources -- Sphinx is a pure-Python package, so should be architecture independent (we don't test on x86 and x64 on Linux, for example)

A